.editorconfig

```diff
@@ -0,0 +1,13 @@
+root = true
+
+[*]
+indent_style = space
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+indent_size = 2
+
+[*.md]
+indent_size = 4
+trim_trailing_whitespace = false
```

Comment:
You should add a break line at the end of the file.

Please check this message: https://automation-bootcamp.slack.com/archives/C017QT1M1DY/p1595723443011200